app/routes/docs.sample.json.md

```diff
@@ -25,7 +25,7 @@ consumer = Consumer(
 )
 
 # Subscribe to Kafka topic
-consumer.subscribe(['igwn.gwalert'])
+consumer.subscribe(['gcn.notices.mission'])
```

Comment:
I think that we should keep this set to `igwn.gwalert`, because it is probably the most popular notice type that contains a HEALPix file.